arcgis-ios-sdk-samples/Analysis/Statistical query (group and sort)/StatisticalQueryGroupAndSortViewController.swift

```diff
@@ -28,9 +28,12 @@ class StatisticalQueryGroupAndSortViewController: UIViewController, UITableViewD
     private var orderByFields = [AGSOrderBy]()
     private var selectedOrderByFields = [AGSOrderBy]()
     private var statisticDefinitions = [AGSStatisticDefinition]()
-    private var sectionHeaderTitles = ["1. Add Statistic Definitions", "2. Select Group By Fields", "3. Select Order By Fields"]
     private var statisticTypes = ["Average", "Count", "Maximum", "Minimum", "StandardDeviation", "Sum", "Variance"]
     
+    private enum Section: Int, CaseIterable {
```

Comment:
Please remove the raw representation. It is no longer needed.
```suggestion
    private enum Section: CaseIterable {
```